### ISSUE TITLE
Nav links need color change

### ISSUE THREAD

**Issue by nfaith** [Tim-Bender/CSCI-331-Project#55]
Issue Description
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

Element location
.nav-item.nav-link[href$="story"]
Element source
<a class="nav-item nav-link" href="/story">Our Story</a>
To solve this issue, you need to...
Fix the following:
Element has insufficient color contrast of 2.57 (foreground color: #007bff, background color: #2d3e73, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Related node:
.footer_color
